src/app/shared/issue-tables/issue-tables.component.ts

```diff
@@ -75,6 +81,18 @@ export class IssueTablesComponent implements OnInit, AfterViewInit {
     });
   }
 
+  sortChange(newSort: Sort) {
+    this.tableSettings.sortActiveId = newSort.active;
+    this.tableSettings.sortDirection = newSort.direction;
+    this.issueTableSettingsService.setTableSettings(this.table_name, this.tableSettings);
+  }
+
+  pageChange(pageEvent: PageEvent) {
+    this.tableSettings.pageSize = pageEvent.pageSize;
+    this.tableSettings.pageIndex = pageEvent.pageIndex;
+    this.issueTableSettingsService.setTableSettings(this.table_name, this.tableSettings);
+  }
+
```

Comment:
I'm not sure if this is the best practice to sync settings of a table component with the service. I would suggest the following:
* Service's settings values to be of type `Observable<TableSettings>`
* Table component settings to sync with service's settings value using `Observable::subscribe`
* In these methods, only call `issueTableSettingsService::setTableSettings`

@CATcher-org/2324s2 What do yall think?

Reply:
In this case wouldn't I still need these two functions to update the TableSettings in the service.
I am not sure of the benefit of the observable implementation since the only things listening and emitting these changes is from the tables component.

Reply:
i don't see the benefit for observables too. I think an observable implementation is beneficial if we have more than 1 table listening to the same set of settings but so far we are saving the settings for each table (or to be more precise `table_name`, but currently each table has a unique `table_name`).